src/test/createRestyleComponent.test.tsx

```diff
@@ -107,12 +111,12 @@ describe('createRestyleComponent', () => {
     it('does not pass styling properties to the child', () => {
       const {root} = render(
         <ThemeProvider theme={theme}>
-          <Component opacity={0.5} pointerEvents="auto" />
+          <Component opacity={0.5} removeClippedSubviews />
```

Comment:
Why was `pointerEvents` replaced by `removeClippedSubviews`?

Reply:
Because pointerEvents is now present in style since rn 0.71 so this test failed. I have used removeClippedSubview but I could choose another property (that is not a style one)